base/Enums.jl

```diff
@@ -46,6 +46,13 @@ julia> f(apple)
 "I'm a Fruit with value: 1"
 ```
 
+Values can also be specified inside a `begin` block, e.g.
+
+    @enum EnumName begin
+        value1
+        value2
+    end
```

Comment:
It would be nice to wrap this in triple backticks with a `julia` annotation so that it will be properly highlighted in the rendered docs.

Reply:
Go ahead :)

Reply:
Okay. I'll add a NEWS item too.

Reply:
thanks!